src/pages/register-a-beacon/activity.tsx

```diff
@@ -76,58 +115,161 @@ const MaritimePleasureOptions: FunctionComponent<OptionsProps> = ({
       <RadioListItem
         id="motor-vessel"
         name={listItemName}
-        value={MaritimePleasureVessel.MOTOR}
+        value={MaritimePleasureType.MOTOR}
         label="Motor vessel"
-        hintText="E.g. Speedboat, RIB"
+        hintText="E.g. Speedboat, Skiff"
         defaultChecked={
-          form.fields.activity.value === MaritimePleasureVessel.MOTOR
+          form.fields.activity.value === MaritimePleasureType.MOTOR
         }
       />
       <RadioListItem
         id="sailing-vessel"
         name={listItemName}
-        value={MaritimePleasureVessel.SAILING}
+        value={MaritimePleasureType.SAILING}
         label="Sailing vessel"
-        hintText="E.g. Skiff, Dinghy, Yacht, Catamaran"
+        hintText="E.g. Dinghy, Yacht, Catamaran"
         defaultChecked={
-          form.fields.activity.value === MaritimePleasureVessel.SAILING
+          form.fields.activity.value === MaritimePleasureType.SAILING
         }
       />
       <RadioListItem
         id="rowing-vessel"
         name={listItemName}
-        value={MaritimePleasureVessel.ROWING}
+        value={MaritimePleasureType.ROWING}
         label="Rowing vessel"
-        hintText="E.g. Single person rowing boat, Cornish Gig, Multi-person rowing boat"
+        hintText="E.g. Rowing boat, Cornish Gig"
         defaultChecked={
-          form.fields.activity.value === MaritimePleasureVessel.ROWING
+          form.fields.activity.value === MaritimePleasureType.ROWING
         }
       />
       <RadioListItem
         id="small-unpowered-vessel"
         name={listItemName}
-        value={MaritimePleasureVessel.SMALL_UNPOWERED}
+        value={MaritimePleasureType.SMALL_UNPOWERED}
         label="Small unpowered vessel"
         hintText="E.g. Canoe, Kayak"
         defaultChecked={
-          form.fields.activity.value === MaritimePleasureVessel.SMALL_UNPOWERED
+          form.fields.activity.value === MaritimePleasureType.SMALL_UNPOWERED
         }
       />
       <RadioListItem
         id="other-pleasure-vessel"
         name={listItemName}
-        value={MaritimePleasureVessel.OTHER}
-        label="Other pleasure vessel"
-        hintText="E.g. Surfboard, Kitesurfing"
+        value={MaritimePleasureType.OTHER}
+        label="Other"
+        hintText="E.g. Surfboard, Kitesurfing, Small punt or tender"
+        defaultChecked={
+          form.fields.activity.value === MaritimePleasureType.OTHER
+        }
+        conditional={true}
+      >
+        <FormGroup errorMessages={form.fields.otherActivityText.errorMessages}>
+          <Input
+            id="otherActivityText"
+            label="Please describe your use"
+            defaultValue={form.fields.otherActivityText.value}
+          />
+        </FormGroup>
+      </RadioListItem>
+    </>
+  );
+};
+
+const MaritimeCommercialOptions: FunctionComponent<OptionsProps> = ({
```

Comment:
Might move these Options List Components into a separate file/files as this page is getting loooonnnggg

Reply:
Yep, agreed. Will do in a later PR when we come to implementing the other activity pages